src/app/home/problem-list/problem-list.component.html

```diff
@@ -1 +1,32 @@
-<p>problem-list works!</p>
+<html>
+  <head>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      th,
+      td {
+        padding: 5px;
+        text-align: left;
+      }
```

Comment:
You can move this to the `problem-list.component.scss` file. All the styles for the component should reside over there.

Reply:
Update in the following commit.
[Commit link](https://github.com/armory-uci/vital/pull/8/commits/066d6038cd7ea68ebd6bcc242bd6fed17dc374fb)